tests/scripts/pulp_python/test_content.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# shellcheck source=tests/scripts/config.source
+. "$(dirname "$(dirname "$(realpath "$0")")")"/config.source
+
+pulp debug has-plugin --name "pulp_python" --min-version "3.1.0.dev" || exit 3
```

Comment:
OK, that might be the reason, the other runs don't fail.
Why do we only allow those tests with 3.1?

And still returning insane output from the CLI is a no no. We might not have seen this before, because shelf reader is the first content with special characters in the header, which let's me to believe this is a general problem.

Reply:
I needed to add some [fixes](https://github.com/pulp/pulp_python/pull/327) to the Python plugin to get the CLI to work.

Reply:
Ah, that's the why only 3.1. Check!